controllers/news.js

```diff
@@ -1,8 +1,20 @@
-const { selectTopics } = require('../models/news');
+const {
+	selectTopics,
+	selectArticlesById,
+	selectArticleComments,
+} = require('../models/news');
 
 exports.getTopics = (req, res) => {
 	selectTopics().then((returnedTopics) => {
 		res.status(200).send(returnedTopics);
-		console.log({ topics: returnedTopics });
+	});
+};
+
+exports.getArticlesById = (req, res) => {
+	const { article_id } = req.params;
+	selectArticlesById(article_id).then((returnedArticle) => {
+		returnedArticle === undefined
+			? res.status(404).send({ msg: 'Route not found' })
```

Comment:
route not found may be slightly misleading as a error message, you also don't want to be doing this in our controller